server/auth/store.go

```diff
@@ -46,6 +46,8 @@ var (
 
 	revisionKey = []byte("authRevision")
 
+	rootPerm = authpb.Permission{PermType: authpb.READWRITE, RangeEnd: []byte{0}}
```

Comment:
can/shell we explicitly set Key to []byte{} for readability ?

Reply:
Fine.